english/wake-up/wake-up-again/wake-up-again.md

```diff
@@ -8,7 +8,7 @@ yet. You were still half-asleep and if you just ignored the sound, you
 
 figured you’d be able to go back to sleep. But not back to the nightmare
 
-you just had, obviously.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.